openpype/cli.py

```diff
@@ -94,6 +94,27 @@ def eventserver(debug,
     )
 
 
+@main.command()
+@click.option("-d", "--debug", is_flag=True, help="Print debug messages")
+@click.option("-e", "--executable", help="Executable")
+@click.option("-u", "--upload_dir", help="Upload dir")
```

Comment:
Add `host` and `port` optional arguments here. It is better to handle those with launch arguments than with environment variables.

Reply:
Makes sense